host/tests/dsHost_L3_Tests/dsHost_test1_VerifyTemperature.py

```diff
@@ -0,0 +1,156 @@
+        # Open Session for player
+        self.player_session = self.dut.getConsoleSession("ssh_player")
+
+        # Open Session for hal test
+        self.hal_session = self.dut.getConsoleSession("ssh_hal_test")
+
+        player = self.cpe.get("test").get("player")
+
+        # Create player Class
+        self.testPlayer = utPlayer(self.player_session, player)
+
+         # Create user response Class
+        self.testUserResponse = utUserResponse()
+
+        # Get path to device profile file
+        self.deviceProfile = dir_path + "/" + self.cpe.get("test").get("profile")
+
+    def testDownloadAssets(self):
+        """
+        Downloads the artifacts and streams listed in test-setup configuration file to the dut.
+
+        Args:
+            None.
+        """
+
+        # List of streams with path
+        self.testStreams = []
+
+        self.deviceDownloadPath = self.cpe.get("target_directory")
+
+        #download test artifacts to device
+        url = self.testSetup.assets.device.test1_EnableDisableAndVerifyAudioPortStatus.artifacts
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+
+        #download test streams to device
+        url = self.testSetup.assets.device.test1_EnableDisableAndVerifyAudioPortStatus.streams
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+            for streampath in url:
+                self.testStreams.append(self.deviceDownloadPath + "/" + os.path.basename(streampath))
+
+    def testCleanAssets(self):
+        """
+        Removes the assets copied to the dut.
+
+        Args:
+            None.
+        """
+        self.deleteFromDevice(self.testStreams)
+
+    def testRunPrerequisites(self):
+        """
+        Runs Prerequisite commands listed in test-setup configuration file on the dut.
+
+        Args:
+            None.
+        """
+
+        #Run test specific commands
+        cmds = self.testSetup.assets.device.test1_EnableDisableAndVerifyAudioPortStatus.execute
+        if cmds is not None:
+            for cmd in cmds:
+                self.writeCommands(cmd)
+
+    def testFunction(self):
+        """This function will test the Audio Ports by enabling and disabling the ports
+
+        Returns:
+            bool
+        """
+
+        # Download the assets listed in test setup configuration file
+        self.testDownloadAssets()
+
+        # Run Prerequisites listed in the test setup configuration file
+        self.testRunPrerequisites()
+
+        # Create the dsHost class
+        self.testdsHost = dsHostClass(self.deviceProfile, self.hal_session)
+
+        self.log.testStart("test1_EnableDisableAndVerifyAudioPortStatus", '1')
+
+        # Initialize the dsHost module
+        self.testdsHost.initialise()
+
+        # Get the CPU temperature
+        self.testdsHost.getCPUTemperature()
+        sleep(60)
+        self.testdsHost.getCPUTemperature()
```

Comment:
How are we verifying the test results ?
after calling the geCPUTemperature twice are we going to check the values we got within threshold ?
this logic is missing in test.

Reply:
Updated with an initial verification. Can discuss further.